caffe2/core/numa.cc

```diff
@@ -6,6 +6,7 @@ CAFFE2_DEFINE_bool(
     "Use NUMA whenever possible.");
 
 #if defined(__linux__) && !defined(CAFFE2_DISABLE_NUMA) && CAFFE2_MOBILE == 0
+#include "/usr/include/numa.h"
```

Comment:
Uhh... I don't know what exactly you're trying to do, but surely this is not the right way to do it.

Reply:
I'd recommend renaming `caffe2/core/numa.h` instead